dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/BootstrapManifest.java

```diff
@@ -150,6 +152,75 @@ public String getBundleVersion() {
         return manifestAttributes.getValue(BUNDLE_VERSION);
     }
 
+    @SuppressWarnings("unchecked")
+    private String calculateSystemPackages(String javaVersion) {
+        try {
+            javaVersion = javaVersion.split("\\.")[0];
+            int version = Integer.parseInt(javaVersion);
+            if (version < 9) {
+                return null;
+            }
+            List<String> packages = new ArrayList<>();
+            Method classGetModule = Class.class.getMethod("getModule"); //$NON-NLS-1$
+            Object thisModule = classGetModule.invoke(getClass());
```

Comment:
I think it would be good to move this code down to line 179, then this whole block is just about getting the methods sorted.